src/univariate/discrete/skellam.jl

```diff
@@ -81,7 +81,14 @@ function cf(d::Skellam, t::Real)
     μ1, μ2 = params(d)
     exp(μ1 * (cis(t) - 1) + μ2 * (cis(-t) - 1))
 end
-#based on SciPy: https://github.com/scipy/scipy/blob/v0.15.1/scipy/stats/_discrete_distns.py
+
+"""
+    cdf(d::Skellam, t::Real)
+
+Implementation based on SciPy: https://github.com/scipy/scipy/blob/v0.15.1/scipy/stats/_discrete_distns.py 
+(OR)
+Refer to Eqn (5) in On an Extension of the Connexion Between Poisson and χ2 Distributions, N.L Johnson, Vol 46, No 3/4(1959), doi:10.2307/2333532 which relates the Skellam and Non-central chisquare PDFs, which is very similar to their CDFs computation as well.
```

Comment:
Some linebreaks?

Reply:
You mean the (OR)?

Reply:
Ah sorry! You meant giving linebreaks in the last line I guess?

Reply:
Done!